pkg/controller/release/release_controller.go

```diff
@@ -435,6 +459,22 @@ func (c *Controller) enqueueRelease(obj interface{}) {
 	c.releaseWorkqueue.Add(key)
 }
 
+func (c *Controller) enqueueReleaseRateLimited(obj interface{}) {
```

Comment:
Not that I mind (as I have already approved it anyway), but these changes are already present in a different PR: #127

Reply:
Yes, some tests fail without it (as they cause updating a release which triggers the gazillion retries)